packages/react/src/utils/useScrollLock.ts

```diff
@@ -49,6 +49,8 @@ function preventScrollStandard(referenceElement: Element | null) {
   }
 
   function lockScroll() {
+    /* DOM reads: */
+
```

Comment:
Grouped reads & writes to avoid this:
![Screenshot From 2025-05-14 03-32-31](https://github.com/user-attachments/assets/cd097a13-bdaa-46f0-a149-e6c81204b429)

Reply:
This is a good start, but floating-ui is also read-then-writing the DOM, so we still have forced reflows, but it will required fixing in or sync'ing with floating-ui. Maybe something like [fastdom](https://github.com/wilsonpage/fastdom).